### DISCUSSION TITLE
arm64 install

### DISCUSSION THREAD

**Discussion by honorkeles** [coinfoundry/miningcore#1471] [General]
anyone tried building MC on an arm64 machine?

**Comment by oliverw** [comment]
You might run into problems with `libmultihash` and `libcryptonight` and the x86 specific optimizations they contain. Not sure. Never tried.

**Reply by honorkeles** [reply]
yeah, first dotnet didn't work then cmake and I got so many errors after figuring out dotnet and cmake. It seems not worth spending time.

**Reply by oliverw** [reply]
> yeah, first dotnet didn't work then cmake and I got so many errors after figuring out dotnet and cmake. It seems not worth spending time.

Sadly, although it would be cool to get it running eventually.
